pkg/idtools/idtools.go

```diff
@@ -101,6 +108,26 @@ type IdentityMapping struct {
 	GIDMaps []IDMap `json:"GIDMaps"`
 }
 
+// FromUserIdentityMapping converts a user.IdentityMapping to an idtools.IdentityMapping
+func FromUserIdentityMapping(u user.IdentityMapping) IdentityMapping {
```

Comment:
This utility is probably only needed for existing code to migrate, correct? Or do we want to continue having the `IdentityMapping` type?

If not, we could consider marking both this function and the type as deprecated, outlining that it's kept to help codebases to transition, but should not be used for new code. Something like;


```suggestion
// FromUserIdentityMapping converts a [user.IdentityMapping] to an [idtools.IdentityMapping].
//
// Deprecated: this is a transitional utility, and should not be used for reasons other than migrating existing code to [user.IdentityMapping].
```

And similar for the type (GitHub doesn't let me comment on those lines);


```go
// IdentityMapping contains a mappings of UIDs and GIDs.
// The zero value represents an empty mapping.
//
// Deprecated: use [user.IdentityMapping]  instead. This type is provided to allow existing code to transition.
type IdentityMapping struct {
	UIDMaps []IDMap `json:"UIDMaps"`
	GIDMaps []IDMap `json:"GIDMaps"`
}
```

Reply:
`idtools.IdentityMapping` is still heavily used, trying to avoid deprecating it in this PR. I was thinking it would be better todo that after splitting archive